frontend/src/hocs/withMapboxToken/MapboxTokenError.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { ReactElement } from "react"
+import ErrorElement from "components/shared/ErrorElement"
+import {
+  MapboxTokenFetchingError,
+  MapboxTokenNotProvidedError,
+} from "hocs/withMapboxToken/MapboxToken"
+
+interface Props {
+  error: Error | MapboxTokenFetchingError | MapboxTokenNotProvidedError
+  deltaType: string
+  width: number
+}
+
+const MapboxTokenError = ({
+  error,
+  width,
+  deltaType,
+}: Props): ReactElement => {
+  if (error instanceof MapboxTokenNotProvidedError) {
+    return (
+      <ErrorElement
+        width={width}
+        name="No Mapbox token provided"
+        message={
+          <>
+            To use `st.{deltaType}` you need to set up a Mapbox access token.
```

Comment:
Use `<code>` instead of backtick.  Also, wrap the message in `<p>` tags to make it more readable:

```
<p>
To use <code>st.{deltaType}</code> you need to set up a Mapbox access token.
</p>

<p>
To get a token, create an account at{" "}<a href="https://mapbox.com">https://mapbox.com</a>. It's free for
moderate usage levels!
</p>

<p>
Once you have a token, just set it using the Streamlit config option <code>mapbox.token</code>. And don't forget to restart your Streamlit server at this point if it's still running, then reload this tab.
</p>

<p>
See{" "}<a href="https://docs.streamlit.io/cli.html#view-all-config-options">our documentation</a>{" "}for more info on how to set config options.
</p>
```

(I also added a line about restarting the Streamlit server and reloading the tab, since this is an issue I ran into just now :wink: )